docs/to_clarify.md

```diff
@@ -18,7 +18,7 @@
 
 1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
 
-    a. What needs to be done during the checking-in process?
+    What needs to be done during the digital checking-in process? Is there any of the hidden/back-end process must be adhered to provide a seamless and contact-less check-in process?
```

Comment:
Why is the bullet point a. removed? 

> Is there any of the hidden/back-end process must be adhered to

What does this mean? As in, what an example of a hidden/back-end process?

Reply:
An unknown step for them to explain. IMO i was thinking of collecting a list of tenants whom are staying currently? or maybe  steps where it is not controllable by software.

Reply:
I don't think it's necessary. As mentioned above https://github.com/openinnovation-hotel-check-in-out/ideation/pull/9#discussion_r464434737, we can remove this point entirely. This question might seem good-to-have for now, but as we clarify User Stories with them, we would necessarily get the answer for this question already. So, let's remove it.